_episodes/02-numpy.md

```diff
@@ -656,38 +656,42 @@ which is the average inflammation per patient across all days.
 
 > ## Change In Inflammation
 >
-> This patient data is _longitudinal_ in the sense that each row represents a
-> series of observations relating to one individual.  This means that
-> the change in inflammation over time is a meaningful concept.
```

Comment:
I am not sure about completely losing this sentence. I like how it set up why it is useful to learn how to use the `diff` function. What are your thoughts on removing it?

Reply:
As mentioned in the [commit message](https://github.com/swcarpentry/python-novice-inflammation/pull/777/commits/20d573e7e828a950b19ed8fe275b533f30ae6f8c):
```
The text part about "patient data is _longitudinal_" is redundant since
that answers the question posed later regarding the usage of the `axis`
keyword. It intoduces the usage of `axis`, while the following intro to
diff() first uses a 1d array.
```
But you are right, it motivates learning about `diff` in the first place, even though the parts about (i) "we can take diffs along axes" and (ii) "taking the diff along axis 1 is useful" is repeated later on. 

In any case, I'd leave it in then since the main point of this PR is the variable name and I have no better introductory line :).